release-scripts/publish-release.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+SCRIPTS_HOME=$(dirname $0)
+
+function help {
+    echo "Usage: $0"
+}
+
+source $SCRIPTS_HOME/check-prereqs.sh
+
+version_target=$(poetry version --short)
+
+echo "Creating git tag for release"
+git tag --sign --message="Release $SUBPROJECT $version_target" $SUBPROJECT-$version_target
+
+
+echo "Pushing tag to origin"
+git push origin $version_target
```

Comment:
Since the tag is now `$SUBPROJECT-$version_target`, shoudn't this be like this?
```suggestion
git push origin $SUBPROJECT-$version_target
```

Reply:
fixed in latest commit